server/server.js

```diff
@@ -1,15 +1,129 @@
 const express = require("express");
 const app = express();
+const cors = require('cors');
 const port = process.env.PORT || 5000;
+const bodyParser = require("body-parser");
+require('dotenv').config();
 
-app.listen(port, () => console.log(`Listening on port ${port}`));
+const corsOptions = {
+  origin: "https://videosfront.onrender.com",
+};
+
+app.use(cors(corsOptions));
+app.use(bodyParser.json());  
+
+const { Pool } = require("pg");
+
+const db = new Pool({
+  user: process.env.USERNAME,
+  host: process.env.HOSTNAME,
+  database: process.env.DATABASE_NAME,
+  password: process.env.PASSWORD,
+  port: 5432,
+});
+
+// let videos = [
+//   {
+//     id: 523523,
+//     title: "Never Gonna Give You Up",
+//     url: "https://www.youtube.com/embed/dQw4w9WgXcQ",
+//     rating: 23,
+//   },
+//   {
+//     id: 523427,
+//     title: "The Coding Train",
+//     url: "https://www.youtube.com/watch?v=HerCR8bw_GE",
+//     rating: 230,
+//   },
+//   {
+//     id: 82653,
+//     title: "Mac & Cheese | Basics with Babish",
+//     url: "https://www.youtube.com/watch?v=FUeyrEN14Rk",
+//     rating: 2111,
+//   },
+//   {
+//     id: 453538,
+//     title:
+//       "The Complete London 2012 Opening Ceremony | London 2012 Olympic Games",
+//     url: "https://www.youtube.com/watch?v=4As0e4de-rI",
+//     rating: 3211,
+//   }
+// ];
+
+// GET "/:{id}"
+app.get("/:id", (req, res) => {
+  let idRequested = Number(req.params.id);
+  db.query('select * from urls where id = $1', [idRequested])
+  .then(result = res.status(200).json(result.rows))
```

Comment:
In general I would expect an endpoint to get `/id` to just return a single object, rather than array containing a single object, and to 404 if the ID was missing - what do you think?